lib/models/hostapp.ts

```diff
@@ -112,14 +113,14 @@ const getHostappModel = function (deps: InjectedDependenciesParam) {
 			// TODO: Don't append the variant and sent it as a separate parameter when requesting a download when we don't use /device-types anymore and the API and image maker can handle it. Also rename `rawVersion` -> `versionWithVariant` if it is needed (it might not be needed anymore).
 			// The version coming from release tags doesn't contain the variant, so we append it here
 			return {
-				id: release.id,
 				osType: appTags.osType,
 				line,
 				strippedVersion: version,
 				rawVersion: `${version}.${normalizedVariant}`,
 				basedOnVersion,
 				variant: normalizedVariant,
 				formattedVersion: `v${version}${lineFormat}`,
+				...release,
```

Comment:
Any reason this isn't on the top, so that we are sure we the custom props that we provide are definitely not overwritten?